From d14732e7069aa8c33733f067e1e706bd852e3aba Mon Sep 17 00:00:00 2001
From: Mike Perry <mikeperry-git@torproject.org>
Date: Tue, 28 Aug 2012 18:05:11 -0700
Subject: [PATCH 15/19] Rebrand Firefox to TorBrowser

This patch does some basic renaming of Firefox to TorBrowser. The rest of the
branding is done by images and icons.
---
 browser/branding/official/configure.sh             |    2 +-
 browser/branding/official/locales/en-US/brand.dtd  |    6 +++---
 .../official/locales/en-US/brand.properties        |    6 +++---
 3 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/browser/branding/official/configure.sh b/browser/branding/official/configure.sh
index 55f3f18..33102b0 100644
--- a/browser/branding/official/configure.sh
+++ b/browser/branding/official/configure.sh
@@ -2,5 +2,5 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-MOZ_APP_DISPLAYNAME=Firefox
+MOZ_APP_DISPLAYNAME=TorBrowser
 MOZ_UA_BUILDID=20100101
diff --git a/browser/branding/official/locales/en-US/brand.dtd b/browser/branding/official/locales/en-US/brand.dtd
index 8e7f6c9..76e405d 100644
--- a/browser/branding/official/locales/en-US/brand.dtd
+++ b/browser/branding/official/locales/en-US/brand.dtd
@@ -2,7 +2,7 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 
-<!ENTITY  brandShortName        "Firefox">
-<!ENTITY  brandFullName         "Mozilla Firefox">
-<!ENTITY  vendorShortName       "Mozilla">
+<!ENTITY  brandShortName        "TorBrowser">
+<!ENTITY  brandFullName         "Tor Browser">
+<!ENTITY  vendorShortName       "Tor Project">
 <!ENTITY  trademarkInfo.part1   "Firefox and the Firefox logos are trademarks of the Mozilla Foundation.">
diff --git a/browser/branding/official/locales/en-US/brand.properties b/browser/branding/official/locales/en-US/brand.properties
index 4a67c55..9ae168e 100644
--- a/browser/branding/official/locales/en-US/brand.properties
+++ b/browser/branding/official/locales/en-US/brand.properties
@@ -2,9 +2,9 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-brandShortName=Firefox
-brandFullName=Mozilla Firefox
-vendorShortName=Mozilla
+brandShortName=TorBrowser
+brandFullName=Tor Browser
+vendorShortName=Tor Project
 
 homePageSingleStartMain=Firefox Start, a fast home page with built-in search
 homePageImport=Import your home page from %S
-- 
1.7.5.4

